rslib/src/browser_table.rs

```diff
@@ -270,7 +271,7 @@ impl RenderContext {
                 } => current_text,
             })
             .join("");
-        let question = extract_av_tags(&qnodes_text, true).0.to_string();
+        let question = extract_av_tags(&qnodes_text, true, &col.tr).0;
```

Comment:
We can now easily implement a separate extract method for the browser, that skips the allocations for AvTags and replaces the tags with something more meaningful instead of "[anki:play:q:0]". But what would that look like?

Reply:
Hmm, that's a bit tricky - if the user has only tts on their front, they'd want to see the text. But if they're using tts to read out a field that's already displayed, it would mean it's shown twice. The former seems a bigger problem than the latter, so unless we want to add some way to customize this, maybe the latter?

Reply:
I agree, and content can always be stripped with a browser appearance template.
But how exactly do we represent a sound or tts tag? Maybe like so?


    [sound:my sound.mp3] [anki:tts lang=en_US]A Sentence.[/anki:tts]
->

    🔉my sound.mp3🔉 💬A Sentence.💬

Or is this distracting and we should stick to ascii?
Either way, we could add an option to customise this for all tags:

    [anki:caution lang=en_US browser_wrapper=⚠]A Sentence.[/anki:caution]
->

    ⚠A Sentence.⚠

Reply:
It's nice and compact, and I don't mind it. There may be some users who don't like it, and may be some who expect to be able to click on the icons. The alternatives I can think of are short ascii (like [S] or wrapping the text in [...]), or just including the content without any special markup like is done with image filenames - I don't have a strong preference for any of the above. I'd resist making it customizable unless there was strong demand for it - we have our hands full already :-)

Reply:
Then I'll go with the emojis for now and see how it's received.